integrations/src/main/scala/com/quantemplate/integrations/qt/QTService.scala

```diff
@@ -0,0 +1,110 @@
+package com.quantemplate.integrations.qt
+
+import io.circe.{Encoder, Decoder}
+import scala.concurrent.{ExecutionContext, Future}
+import org.slf4j.LoggerFactory
+import akka.actor.typed.ActorSystem
+import cats.syntax.option.given
+import akka.http.scaladsl.model.{HttpEntity, FormData}
+import akka.http.scaladsl.model.headers.{ Authorization, OAuth2BearerToken }
+
+import com.quantemplate.integrations.common.{View, Config, HttpService}
+import com.quantemplate.integrations.common.HttpService.*
+import com.quantemplate.integrations.qt.QTModels.*
+
+
+class QTService(httpService: HttpService)(using ec: ExecutionContext, conf: Config.Quantemplate):  
+  lazy val logger = LoggerFactory.getLogger(getClass)
+  lazy val endpoints = QTEndpoints(conf.api.baseUrl)
+
+  def executePipeline(orgId: String, pipelineId: String) =
+    val endpoint = endpoints.executions(orgId, pipelineId)
+
+    for
+      auth <- getToken().map(getTokenAuthHeaders)
+      res <- httpService.post[PipelineExecutionResponse](endpoint, HttpEntity.Empty, auth)
+
```

Comment:
not needed empty line .

I hate writing comments like that.

btw, it looks like scalafmt for scala 3 is available:
https://github.com/scalameta/scalafmt/issues/2216

could you try to attach it to the project?

Reply:
Yes, I will try to add it in a subsequent PR